src/js/router.coffee

```diff
@@ -1,4 +1,21 @@
 class Router extends Backbone.Router
+
+
+  # Set Router.navigateAwayMessage to a string to confirm when a user is navigating
+  # away from their current route. Set it to false to turn off the confirmation.
+  navigateAwayMessage: false
+
+  # Override Backbone.Router.execute
```

Comment:
@chrisekelley pointed out the potential use of monitoring window event `popstate`. Because this strategy is suggested by the Backbone docs, I'm going to guess we might experience less bugs this way because Backbone is expecting us to use this method (right? not always). 

We do have a listener for `popstate` here https://github.com/Tangerine-Community/Tangerine-client/commit/a110bdad18f1daa19eac7082be308508c59e29b9#diff-09f368622c61426994f4a61515e1e0baR235